frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue

```diff
@@ -24,7 +24,8 @@
         class="mini-composer__circuit-section__reset"
         @click="onResetButton"
       >
-        Reset <PlayIcon />
+
+      Reset <ResetIcon class="mini-composer__circuit-section__reset__icon" />
```

Comment:
Here we need to remove the empty line and add some indentation:

```suggestion
        Reset <ResetIcon class="mini-composer__circuit-section__reset__icon" />
```